specification/data-database.md

```diff
@@ -1,32 +1,75 @@
-# Semantic conventions for database client calls
+# Database (SQL and NoSQL) client calls
+
+> 🚧 WORK IN PROGRESS
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Connection-level attributes](#connection-level-attributes)
+- [Call-level attributes](#call-level-attributes)
+- [Notes for specific database types](#notes-for-specific-database-types)
+
+<!-- tocstop -->
 
 For database client call the `SpanKind` MUST be `Client`.
 
 Span `name` should be set to low cardinality value representing the statement
-executed on the database. It may be stored procedure name (without argument), sql
+executed on the database. It may be stored procedure name (without argument), SQL
 statement without variable arguments, etc. When it's impossible to get any
 meaningful representation of the span `name`, it can be populated using the same
-value as `db.instance`.
+value as `db.name`.
+
+## Connection-level attributes
 
-Note, Redis, Cassandra, HBase and other storage systems may reuse the same
-attribute names.
+These attributes will usually be the same for all operations performed over the same database connection
```

Comment:
>These attributes will usually be the same for all operations performed over the same database connection although some database systems may allow to switch to a different db.user within the same connection and other database systems may not even have the concept of a connection.

These attributes will usually be the same for all operations performed over the same database connection. Some database systems may allow a connection to switch to a different `db.user`, and other database systems may not even have the concept of a connection.

Reply:
Thanks for the suggestions, I'll apply them. Just a hint for the future: You can use the "Suggest changes button (the +/- thingy over the text field) so that a diff appears in your comment. That makes it easier to spot the difference between the current and suggested wording.

Reply:
I would have, but some of these spanned multiple lines and the GitHub support for that is lacking